.env.dev

```diff
@@ -14,8 +14,8 @@ REACT_APP_UNCHAINED_AVALANCHE_HTTP_URL=https://dev-api.avalanche.shapeshift.com
 REACT_APP_UNCHAINED_AVALANCHE_WS_URL=wss://dev-api.avalanche.shapeshift.com
 REACT_APP_UNCHAINED_OPTIMISM_HTTP_URL=https://dev-api.optimism.shapeshift.com
 REACT_APP_UNCHAINED_OPTIMISM_WS_URL=wss://dev-api.optimism.shapeshift.com
-REACT_APP_UNCHAINED_BNBSMARTCHAIN_HTTP_URL=https://dev-api.bnbsmartchain.shapeshift.com
-REACT_APP_UNCHAINED_BNBSMARTCHAIN_WS_URL=wss://dev-api.bnbsmartchain.shapeshift.com
+REACT_APP_UNCHAINED_BNBSMARTCHAIN_HTTP_URL=https://api.bnbsmartchain.shapeshift.com
```

Comment:
How temporary is this? Did we want to undo before merging this PR?

Reply:
nah, dev still borked and want to get some testing rolling. I will be sure to revert once stable again